personio-personnel-data-api-oa3.yaml

```diff
@@ -43,6 +43,23 @@ paths:
         - BearerAuth: []
       tags:
         - Employees
+      parameters:
+        - name: limit
+          in: query
+          required: false
+          description: Pagination attribute to limit how many employees are per page
+          schema:
+            type: integer
+            minimum: 1
+            default: 200
+        - name: offset
+          in: query
+          required: false
+          description: Pagination attribute to identify which page you are requesting, by the form of telling an offset from the first record that would be returned.
```

Comment:
Pagination attribute to identify the cursor for the pagination in combination with the limit attribute. For example, given a limit of 100 and an offset of 1, the response will skip the first 100 and return the next 100 employees.

Reply:
I actually took that description from description of offset field from other APIs, it looks a bit fuzzy but should we keep it consistent for now?

Reply:
Well, to be honest, we shouldn't ... for me as a reader, I don't quite understand the sentence ... Don't know who is responsible for those texts ... but since they are public they should be signed off by the proper teams (if we have such a team).

Reply:
That sounds reasonable, I updated those texts.

jfyi, `offset = 0` and `offset = 1` behave the same, it always returns the first page. I know it is weird behavior but again other APIs are behaving that way thus I kept the same behavior.